client/components/LevelOne.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const LevelOne = (props) => {
+  const {
+    nextTwoWeeks, times, max, next, change, count, date, time,
+  } = props;
+
+  const guestCount = [];
+  for (let j = 1; j <= 12; j += 1) {
+    if (j <= max) {
+      guestCount.push(j);
+    } else {
+      break;
+    }
+  }
+
+  return (
+    <div>
+      <h3>
+        Make a Reservation
+      </h3>
+      <div>
+        <select onChange={change} name="selectedDate" value={date}>
+          {nextTwoWeeks.map(e => (
+            <option value={e}>
+              {e}
+            </option>
+          ))}
+        </select>
+      </div>
+      <div>
+        <select onChange={change} name="selectedTime" value={time}>
+          {times.map(ele => (
```

Comment:
Assuming ele means a time, you might call it time.

Reply:
Same as above.